cmd/etcd-launcher/main.go

```diff
@@ -75,8 +81,27 @@ func main() {
 	log := rawLog.Sugar()
 	log = log.With("member-name", e.config.podName)
```

Comment:
I think this is not really needed, no? The pod's name will be attached to the logs anway whenervr someone picks up the logs (fluentbit, promtail). Having it explicitly here just clutters the log, no?

Reply:
good point.